apps/src/sites/studio/pages/devise/sessions/_login.js

```diff
@@ -13,8 +15,12 @@ $(document).ready(() => {
     false
   );
   const signupLink = document.getElementById('signup-link');
+  const {userReturnTo} = getScriptData('data');
```

Comment:
Awesome! my only nitpick would be `data` seems a bit too generic. maybe `login` since it's the login page? or even just `userReturnTo` and remove the destructuring here?

I just looked and a bunch of pages use `props` as the script data wrapper 🙄 ... but I imagine there could be issues with naming collisions eventually if we continue that

![Screenshot 2024-10-31 at 3 41 50 PM](https://github.com/user-attachments/assets/4283a7b1-375e-4e5b-aa8b-a027289b5481)

Reply:
very fair- I forgot I could just convert the path to json. Was able to test that the destructuring worked!